core/src/main/java/hera/core/commands/Command.java

```diff
@@ -5,10 +5,20 @@
 import discord4j.core.object.entity.Guild;
 import discord4j.core.object.entity.Member;
 import discord4j.core.object.entity.MessageChannel;
+import hera.database.types.CommandName;
 import reactor.core.publisher.Mono;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public interface Command {
+
+    Map<CommandName, Command> COMMANDS = new HashMap<>();
```

Comment:
this should be in a diffrent file in my opinion

Reply:
Moved it to its own class